index.bs

```diff
@@ -764,12 +764,35 @@ partial interface HTMLIFrameElement {
     <p>To get the <a>observable policy</a> for an Element |node|, run the
     following steps:</p>
         1. Let |inherited policy| be an empty [=ordered map=].
-        3. [=set/For each=] <a>supported feature</a> |feature|:
+        2. [=set/For each=] <a>supported feature</a> |feature|:
             1. Let |isInherited| be the result of running <a abstract-op>Define
                 an inherited policy for feature in container at origin</a> on
                 |feature|, |node| and |node|'s <a>declared origin</a>.
             2. Set |inherited policy|[|feature|] to |isInherited|.
-        4. Return a new <a>permissions policy</a> with <a for="permissions
+            3. If |node| is a <a>navigable container</a>:
```

Comment:
This code is good - it should probably be somewhere else, though, looking at it again. This is a side effect attached to the getter for the observable policy, which would mean that sending reports would be triggered by a script calling `element.featurePolicy,allowedFeaures()` or similar methods.

Is the intention here to have this called when an iframe is created, or when its src/allow attribute changes?